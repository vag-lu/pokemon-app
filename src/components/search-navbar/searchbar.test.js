import { shallow, configure } from 'enzyme';
import SearchNavbar from '.';
import React from 'react'
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('SearchNavbar', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SearchNavbar />);
    expect(wrapper).toMatchSnapshot();
  });
});