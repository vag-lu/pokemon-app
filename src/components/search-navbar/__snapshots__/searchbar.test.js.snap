// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton renders correctly 1`] = `ShallowWrapper {}`;

exports[`SearchNavbar renders correctly 1`] = `ShallowWrapper {}`;
